recipes/mtspec/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.3.0" %}
+{% set sha256 = "307dc34acefdfae06f055d99d0260cb3529a1fd46d7696be2ab533752aff01a1" %}
+
+package:
+  name: mtspec
+  version: {{ version }}
+
+
+source:
+  fn: mtspec-{{ version }}.zip
+  url: https://pypi.io/packages/source/m/mtspec/mtspec-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
`Fortran` extensions on Windows are possible, but complicated. I would skip the Windows build for now by adding:

```yaml
skip: True  # [win]
```

PS: Once the `feedstock` is created I can help you with a Windows build.